docs/how-to/download-new-upstream-version.rst

```diff
@@ -1,9 +1,139 @@
 ..  _download-new-upstream-version:
 
-===============================
-Download a new Upstream Version
-===============================
+Download a new :term:`Upstream` version
+=======================================
 
 .. caution::
 
     Work in progress!
+
+Once in a while you need to download a new :term:`Upstream` release or check if a newer
+:term:`Upstream` release exists.
+
+Most of the :term:`Source Packages <Source Package>` contain a ``watch`` file in the ``debian``
+folder. This is a configuration file for the :manpage:`uscan(1)` utility which allows you to
+automatically search :term:`HTTP` or :term:`FTP` sites or :manpage:`git(1)` repositories
+for newly available updates of the :term:`Upstream` project.
+
+.. note::
+    If the :term:`Source Package` does not contain a :file:`debian/watch` file, there may be
+    an explanation and instructions in the :file:`debain/README.source` or
+    :file:`debian/README.debian` file (if available) that tell you how to proceed.
+
+.. caution::
```

Comment:
This caution box is quite large when rendered on the page, and it seems to be mostly relevant to the download section. Can we incorporate this cautionary content into the instructions in that section?

Reply:
@s-makin What do you think of converting the caution box into a section "Best practices"

Reply:
I think that's a great idea!